process/block/baseProcess.go

```diff
@@ -1288,6 +1288,35 @@ func (bp *baseProcessor) DecodeBlockBody(dta []byte) data.BodyHandler {
 	return body
 }
 
+func (bp *baseProcessor) saveEpochStartInfoToStaticStorage(header data.HeaderHandler, marshalledHeader []byte, body *block.Body) {
+	epochStartBootstrapKey := append([]byte(common.EpochStartStaticBootstrapKeyPrefix), []byte(fmt.Sprint(header.GetEpoch()))...)
+	err := bp.store.Put(dataRetriever.EpochStartStaticUnit, epochStartBootstrapKey, marshalledHeader)
+	if err != nil {
+		log.Warn("saveEpochStartInfoToStaticStorage.Put header", "error", err.Error())
+	}
+
+	log.Debug("saveEpochStartInfoToStaticStorage: put metaBlock into epochStartStaticStorage", "epoch", header.GetEpoch())
+
+	for i := 0; i < len(body.MiniBlocks); i++ {
+		if body.MiniBlocks[i].Type != block.PeerBlock {
+			continue
+		}
+
+		marshalizedMiniBlock, err := bp.marshalizer.Marshal(body.MiniBlocks[i])
+		if err != nil {
+			log.Warn("saveEpochStartInfoToStaticStorage.Marshal", "error", err.Error())
```

Comment:
missing continue statement?

if marshal fails, why should we continue with `Put` operation?

Reply:
added continue statements